.gitignore

```diff
@@ -19,4 +19,5 @@ pkg
 *.state-*
 *.state
 
-Cargo.lock
\ No newline at end of file
+Cargo.lock
+/src/v1/generated/google.protobuf.rs
```

Comment:
Could you explain why this should be ignored? Don't we depend on it?

Reply:
Yes, because we don't set this https://docs.rs/tonic-build/latest/tonic_build/struct.CodeGenBuilder.html#method.compile_well_known_types

Meaning that google.protobuf types are not compiled and instead the Prost provided ones are used.

Reply:
I added a comment to gitignore.